src/sdk/namespace_fs.js

```diff
@@ -889,10 +889,12 @@ class NamespaceFS {
         try {
             let dir = path.dirname(obj);
             while (dir !== this.bucket_path) {
-                const entries = await nb_native().fs.readdir(fs_account_config, dir);
-                if (entries.length === 0) {
+                try {
```

Comment:
```javascript
// not allowing keys with dots which can be treated as relative paths and "leave" the bucket_path
if (key.includes('.')) throw new Error('Bad relative path key ' + key);
```
this will fail when we have a key with a file extension (for example `file_name.tar`)
what about:
```javascript
// not allowing keys with dots follow by slash which can be treated as relative paths and "leave" the bucket_path
if (key.includes('./')) throw new Error('Bad relative path key ' + key);
```